### DISCUSSION TITLE
How to update `-alpha` packages together with the stable packages family?

### DISCUSSION THREAD

**Discussion by moltar** [renovatebot/renovate#21268] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

AWS CDK project comes with many `-alpha` grade packages, but they are usually released under the same pattern of semver as the stable part of the project.

Example:

```json
"aws-cdk-lib": "2.72.1",
"@aws-cdk/aws-apigatewayv2-alpha": "2.72.1-alpha.0",
"@aws-cdk/aws-apigatewayv2-authorizers-alpha": "2.72.1-alpha.0",
```

Now Renovate seems to ignore the alpha packages entirely.

I want to update the automatically, but ideally also together with the stable package.

Is this possible?

### What you tried so far.

Nothing in particular.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
check the ignore unstable config option

**Reply by moltar** [reply]
Thanks for the pointer.

Here's what I got now:

```json
 "packageRules": [
    {
      "groupName": "aws cdk alpha",
      "ignoreUnstable": false,
      "matchPackagePatterns": ["^@aws-cdk/"]
    }
  ]
```
